geo/src/lib.rs

```diff
@@ -177,9 +177,25 @@
 //!
 //! The following optional [Cargo features] are available:
 //!
-//! - `proj-network`: Enables [network grid] support for the [`proj` crate]. After enabling this feature, [further configuration][proj crate file download] is required to use the network grid
-//! - `use-proj`: Enables coordinate conversion and transformation of `Point` geometries using the [`proj` crate]
-//! - `use-serde`: Allows geometry types to be serialized and deserialized with [Serde]
+//! - `i_overlay`:
+//!     - Enables the `i_overlay` crate, which provides boolean operations on geometries.
```

Comment:
Can we call this feature `bool_ops`? I think it will be more meaningful to our users.

Reply:
Assuming we won't have a separate boolean operations implementation in the future that doesn't depend on `i_overlay`

Reply:
Given how long it's taken to get to this point, that seems unlikely. Not impossible, but a small risk IMO.

Reply:
In case my original concern wasn't clear: I think a user perusing the feature flags will more quickly understand what a `bool_ops` feature does, vs. an `i_overlay` feature. 

The fact that we're using `i_overlay` for bool_ops is an implementation detail that the user probably doesn't care about.

Reply:
Cool I'll change it to `bool_ops`

Reply:
Nevermind, I'm just going to remove the feature